MAINTAINERS.yml

```diff
@@ -227,6 +226,25 @@ Bluetooth Mesh:
         - "area: Bluetooth Mesh"
         - "area: Bluetooth"
 
+Bluetooth Audio:
+    status: maintained
+    maintainers:
+        - Thalley
+        - asbjornsabo
+    collaborators:
+        - Vudentz
+        - jhedberg
+        - Casper-Bonde-Bose
+        - MariuszSkamra
+        - rymanluk
+    files:
+        - subsys/bluetooth/audio/
```

Comment:
remove this entry from the top level Bluetooth area and add it to files-exclude, this way we will not have any duplication.

Reply:
Do you know how the labels work? Should `- "area: Bluetooth"` be avoid in the Bluetooth Audio entry?

Reply:
you can still add it, thats fine.